tests/test_config_ssot.py

```
@@ -0,0 +1,237 @@
+        from backend.constants import get_variant_config, reload_constants
+
+        reload_constants()  # Ensure fresh load
+        variants = get_variant_config()
+
+        # Check expected variants exist
+        assert "premium" in variants
+        assert "cheap" in variants
+        assert "local" in variants
+
+        # Check each variant has required fields
+        for name, config in variants.items():
```

Ruff (0.14.8)

Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)